package.json

```diff
@@ -47,19 +48,20 @@
     "eslint-plugin-prettier": "^3.1.1",
     "eslint-plugin-react": "^7.16.0",
     "eslint-plugin-react-hooks": "^2.2.0",
+    "history": "^4.10.1",
```

Comment:
Why we don't use native History API?

Reply:
We need to support server-side apps and the native history is not available in this mode. Also, this module used by react-router package and does not include more bytes to the final app.